New assembler flag

jpaganini opened the pull request:
Added new flag '-a' for specifying assembler tool. Only unicycler or spades are available. 

If 'unicycler' is selected, coverage will not be used as a criteria for filtering potential contamination, since Unicycler already does a similar step.

duboism commented on plaScope.sh:
```diff
@@ -22,7 +22,7 @@
 ##############################################################
 
-VERSION=1.3.1
+VERSION=1.3
```
Why did you change this ?

duboism replied:
OK, I see this was reverted in 82657605a96294a93b6664e53f13ef95efe6e053.

duboism commented:
I don't have an opinion on the utility of the modification from a bioinformatics point of view.

However, the current proposal breaks compatibility because the `-a` option is mandatory. It's not a problem _per se_ bur require some thoughts on the versioning: keeping `-a` mandatory would require switching to version 2.0; another possibility is to use `spades` as the default value.

jpaganini commented:
Hi Mathieu,

I completely agree with using spades as default option in mode 2. I just modified that.

I also modified the README file.

Let me know if addtional modificaitons are required.

Cheers!

duboism commented:
Hello,

Sorry for the delay. We discussed some ideas about your contribution but I think we will merge it as-this quickly. We just want to review the `README.md` file before doing a new version. 

Just a little task for you: we have added a `Contributors` section in `README.md`; Could you merge `master` in your branch and add a line to describe your work in this section ?

jpaganini commented:
Hi,

Thanks for the answer. I've been a bit busy myself lately. Modifications done!

duboism commented:
Hello,

I can't see the commit(s) for your contribution. Did you push ?

jpaganini commented:
Sorry! I havent! It should be present now!

duboism commented:
@GuilhemRoyer : if you're OK, I can merge this.

GuilhemRoyer commented:
> Thanks !
> 
> @GuilhemRoyer : if you're OK, I can merge this.

Thank you @duboism, for sure I am Ok. Thank you also @jpaganini for your contribution.

duboism commented:
I have merged the branch. Thanks @jpaganini ! I will release a new version as soon as we have decided for #5 .
